balancer-js/examples/join.ts

```diff
@@ -0,0 +1,67 @@
+import dotenv from 'dotenv';
```

Comment:
As discussed -
- fix incorrect param inputs
- update to use local fork
- also if we can change to use the full SDK interface so something like:
```
const balancer = new BalancerSDK(...);
await balancer.poolsProvider.fetchPools();
const pool = await balancer.poolsProvider.find(poolId)
if (!pool) throw new Error()
const { data } = pool.buildJoin(params)
```

Reply:
Done  ✅  - with the exception that instead of fetching all pools the provider only fetches a single pool on find(poolId)